docs/addons/container-insights.md

```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
+
+1. Open the CloudWatch console and choose the cluster that you want to view logs for. The log group name format is /aws/eks/<cluster-name>/cluster.
+2. Choose the log stream to view. The following list describes the log stream name format for each log type.
+   1. Kubernetes API server component logs (api) – kube-apiserver-<nnn...>
+   2. Audit (audit) – kube-apiserver-audit-<nnn...>
+   3. Authenticator (authenticator) – authenticator-<nnn...>
+   4. Controller manager (controllerManager) – kube-controller-manager-<nnn...>
+   5. Scheduler (scheduler) – kube-scheduler-<nnn...>
+
+Next in the console, click on Log groups under Logs. 
+
+You will see under log streams all the log streams from your Amazon EKS control plane. 
+
+(insert image)
+
+
+## View workload level logs
+
+In order to view workload level logs follow these steps after browsing to the CloudWatch Logs Insights console
+
+In the navigation pane, choose Insights.
+
+Near the top of the screen is the query editor. When you first open CloudWatch Logs Insights, this box contains a default query that returns the 20 most recent log events.
+
+In the box above the query editor, select one of the Container Insights log groups to query. For the following example queries to work, the log group name must end with performance.
+
+When you select a log group, CloudWatch Logs Insights automatically detects fields in the data in the log group and displays them in Discovered fields in the right pane. It also displays a bar graph of log events in this log group over time. This bar graph shows the distribution of events in the log group that matches your query and time range, not only the events displayed in the table.
+
+In the query editor, replace the default query with the following query and choose Run query.
+
+STATS avg(node_cpu_utilization) as avg_node_cpu_utilization by NodeName
+`SORT avg_node_cpu_utilization DESC`
+This query shows a list of nodes, sorted by average node CPU utilization.
+
+To try another example, replace that query with another query and choose Run query. More sample queries are listed later on this page.
+
+STATS avg(number_of_container_restarts) as avg_number_of_container_restarts by PodName
+`SORT avg_number_of_container_restarts DESC`
+This query displays a list of your pods, sorted by average number of container restarts.
+
+If you want to try another query, you can use include fields in the list at the right of the screen. For more information about query syntax, see CloudWatch Logs Insights Query Syntax.
+
+## View metrics for cluster and workloads
+Under Performance monitoring, the Container Insights dashboard allows you to hone in on both cluster and workload metrics. After selecting EKS Pods and Clusters, you will see that the dashboard provides CPU and memory utilization along with other important metrics such as network performance. 
+
+## View containers via that container map in container insights.
```

Comment:
Let's move this section to the top with the picture. It provides high level insights. The details then follow.

Reply:
Done